### ISSUE TITLE
[Missing SNMP Polling Profile] Moxa NPort 5650-8

### ISSUE THREAD

**Issue by cstoli-92** [kentik/snmp-profiles#448]
[NOTE]: # ( ^^ Please add the Device Vendor and Model in the title above. ^^ )

Device Vendor: Moxa

Device Model: NPort 5650-8

[NOTE]: # (If you do not know your SysOID, it can be gathered from the SNMP walk you provide at the end of this issue. It is the value of this OID in the SYSTEM-MIB: `1.3.6.1.2.1.1.2.0`)

System Object Identifier (SysOID): .1.3.6.1.4.1.8691.2.7

List of critical metrics that you expect in the profile (does not need to be all-inclusive, just a general idea of your expectations):

 - CPU stats
 - Power supply status
 - Interface status
 
If there is a New Relic technical support case related to this request, please provide the case number:

**Please provide a [public gist](https://gist.github.com/) with a sanitized** *(sensitive information removed)* **SNMP walk of the device. Providing MIB files is not a substitute for an SNMP walk and failure to provide one here will result in delays for closing this issue. The output of these commands shows the "true story" of what a device will respond to in SNMP and is used to ensure that our profiles are efficient and accurate for your devices.**

Gist Link: https://gist.github.com/cstoli-92/d484c5431e98288a2045def4553e5123

---
Example SNMP walk commands:
```
# v2c example
snmpwalk -v 2c -t 10 -On -c $COMMUNITY_STRING $IP_ADDRESS . >> snmp_walk.out

# v3 example
snmpwalk -v3 -t 10 -l authPriv -u $USERNAME -a MD5|SHA -A $AUTH_PASS -x DES|AES -X $PRIV_PASS -ObentU -Cc $IP_ADDRESS . >> snmp_walk.out
```

**Comment by thezackm** [comment]
@cstoli-92 - your provided SNMP walk looks to have timed out in the middle of the `SNMPv2-MIB` 

```
.1.3.6.1.2.1.11.32.0 = Counter32: 0
```
We expect a fully completed walk to have much more information, and the majority of devices will have their most valuable performance metrics in the private side of the MIB tree (`.1.3.6.1.4.*`)

Can you try again and increase the timeout value please?

```
# v2c example
snmpwalk -v 2c -t 30 -On -c $COMMUNITY_STRING $IP_ADDRESS . >> snmp_walk.out

# v3 example
snmpwalk -v3 -t 30 -l authPriv -u $USERNAME -a MD5|SHA -A $AUTH_PASS -x DES|AES -X $PRIV_PASS -ObentU -Cc $IP_ADDRESS . >> snmp_walk.out
```

If that doesn't work, you can try targeting the private MIB space directly:
```
# v2c example
snmpwalk -v 2c -t 30 -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4 >> snmp_walk.out

# v3 example
snmpwalk -v3 -t 30 -l authPriv -u $USERNAME -a MD5|SHA -A $AUTH_PASS -x DES|AES -X $PRIV_PASS -ObentU -Cc $IP_ADDRESS .1.3.6.1.4 >> snmp_walk.out
```

**Comment by cstoli-92** [comment]
I've updated the gist with the new contents. Although I don't think it's any different. The second option you provided has even less results.

**Comment by Mesverrum** [comment]
@cstoli-92 I did some poking around on the internet and these Moxa serial adapters don't appear expose any resource utilization metrics at all, just the IF-MIB and the TCP-MIB.  Our default profile already contains the IF-MIB and we have opted not to include the TCP MIB in most profiles because it has a very poor signal to noise ratio, lots of data but it doesn't really help our users troubleshoot problems to justify the expense of storing it all.  These devices do not need a special profile as there is nothing that they offer in SNMP that we don't already get from the base.
